README.md

```diff
@@ -28,15 +28,17 @@
 
 The best place to start is the [Android Chat Tutorial](https://getstream.io/tutorials/android-chat/#java). It teaches you how to use this SDK and also shows how to make frequently required changes. You can use either [Java](https://getstream.io/tutorials/android-chat/#java) or [Kotlin](https://getstream.io/tutorials/android-chat/#kotlin) depending on your preference.
 
-## Clone the Github Example App
+## Sample App
 
-This repo includes a fully functional example app. To run the example app:
+This repo includes a fully functional example app featuring threads, reactions, typing indicators, optimistic UI updates and offline storage.
```

Comment:
>optimistic

What does this mean?

Reply:
![image](https://user-images.githubusercontent.com/265409/97727559-2ac67d80-1a96-11eb-98e9-5c691b118dd2.png)

TL/DR update the UI first, wait for API call result after...

Reply:
Ah, okay, never came across this before. Makes sense 👍